packages/renderer/src/lib/extensions/ExtensionDetails.svelte

```diff
@@ -63,10 +66,32 @@ $: extension = derived(
     <div slot="detail" class="flex">
       <ExtensionBadge class="mt-2" extension="{$extension}" />
     </div>
+    <!-- Display tabs only if extension is in error state -->
+    <svelte:fragment slot="tabs">
+      {#if $extension.state === 'failed'}
+        <Button
+          type="tab"
+          on:click="{() => {
+            screen = 'README';
+          }}"
+          selected="{screen === 'README'}">Readme</Button>
+        <Button
+          type="tab"
+          on:click="{() => {
+            screen = 'ERROR';
+          }}"
+          selected="{screen === 'ERROR'}">Error</Button>
+      {/if}
+    </svelte:fragment>
+
     <svelte:fragment slot="content">
       <div class="flex w-full h-full overflow-y-auto p-4 flex-col lg:flex-row">
-        <ExtensionDetailsSummaryCard extensionDetails="{$extension}" />
-        <ExtensionDetailsReadme readme="{$extension.readme}" />
+        {#if screen === 'README'}
```

Comment:
Why not using navigiation url here ? instead of storing the state in a variable

Reply:
because url is the same